src/components/Migration/MigrationModal.tsx

```diff
@@ -208,8 +208,8 @@ function MigrationModal({ open, handleClose }: { open: boolean; handleClose: any
             aria-label="payout token tabs"
             className="payout-token-tabs"
           >
-            <Tab label={`gOHM`} className="payout-token-tab" />
             <Tab label={`sOHM`} className="payout-token-tab" />
+            <Tab label={`gOHM`} className="payout-token-tab" />
```

Comment:
need to also change the `isGohm` check 
![image](https://user-images.githubusercontent.com/80423742/149988700-02824e3c-4f23-45bb-b2b0-df0fa2827223.png)

Reply:
https://github.com/OlympusDAO/olympus-frontend/blob/a6567fa740693eb6ebecd303458549c845e6c8ba/src/components/Migration/MigrationModal.tsx#L125